### PR TITLE
Update outdated CLI edit releases information

### LINKED ISSUE #30245: Outdated Information for GitHub CLI on Releases
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository

### What part(s) of the article would you like to see updated?

Update the section that states the GitHub CLI does not support editing releases when it now does.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by timsearle**
### Why:

The website documentation is outdated - GitHub CLI does support editing releases.

Closes: #30245

### What's being changed (if available, include any code snippets, screenshots, or gifs):

Following the convention in the "Create releases" section, show an example and a link to the GitHub CLI manual for the command:

Screenshot 2023-11-29 at 19 38 51

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline.

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).

### CODE REVIEW

**Comment by steveward** [comment]
Thanks for the PR @timsearle! I'll get this triaged for review. ⚡

File: content/repositories/releasing-projects-on-github/managing-releases-in-a-repository.md [hunk 1]
```diff
@@ -79,7 +79,13 @@ If you @mention any {% data variables.product.product_name %} users in the notes
 
 {% cli %}
 
-Releases cannot currently be edited with {% data variables.product.prodname_cli %}.
+1. To edit a release, use the `gh release edit` subcommand. Replace `tag` with the tag representing the release you wish to edit. For example, to edit a given release's title:
```
Comment (sabrowning1) [inline content/repositories/releasing-projects-on-github/managing-releases-in-a-repository.md:82]:
```suggestion
1. To edit a release, use the `gh release edit` subcommand. Replace `TAG` with the tag representing the release you wish to edit. For example, to edit a given release's title, use the following code:
```

Just a couple small tweaks for ease of localization, and to adhere to our style guide.

File: content/repositories/releasing-projects-on-github/managing-releases-in-a-repository.md [hunk 2]
```diff
@@ -79,7 +79,13 @@ If you @mention any {% data variables.product.product_name %} users in the notes
...
+
+   ```shell
+   gh release edit <tag> -t "New title"
```
Comment (sabrowning1) [inline content/repositories/releasing-projects-on-github/managing-releases-in-a-repository.md:85]:
```suggestion
   gh release edit TAG -t "New title"
```

One tiny change to align the formatting with our [style guide](https://docs.github.com/en/contributing/style-guide-and-content-model/style-guide#placeholders).

File: content/repositories/releasing-projects-on-github/managing-releases-in-a-repository.md [hunk 3]
```diff
@@ -79,7 +79,13 @@ If you @mention any {% data variables.product.product_name %} users in the notes
...
+   ```
+
+   For more information about possible arguments, see [the {% data variables.product.prodname_cli %} manual](https://cli.github.com/manual/gh_release_edit)
```
Comment (sabrowning1) [inline content/repositories/releasing-projects-on-github/managing-releases-in-a-repository.md:88]:
```suggestion
   For more information about possible arguments, see [the {% data variables.product.prodname_cli %} manual](https://cli.github.com/manual/gh_release_edit).
```

Small typo fix.

**Review by sabrowning1 — CHANGES_REQUESTED** [review]
👋🏼 @timsearle, thanks so much for catching this and raising a PR to update the docs! I've pushed a small formatting fix and left a few small comments below. Other than that, this is looking great! Once those suggestions have been applied, I'm happy to get this merged 🙂

**Comment by timsearle** [comment]
> 👋🏼 @timsearle, thanks so much for catching this and raising a PR to update the docs! I've pushed a small formatting fix and left a few small comments below. Other than that, this is looking great! Once those suggestions have been applied, I'm happy to get this merged 🙂

Thanks @sabrowning1 - especially for the commit suggestions, made it a lot quicker. Should all be done now.

File: content/repositories/releasing-projects-on-github/managing-releases-in-a-repository.md [hunk 4]
```diff
@@ -79,7 +79,13 @@ If you @mention any {% data variables.product.product_name %} users in the notes
 
 {% cli %}
 
-Releases cannot currently be edited with {% data variables.product.prodname_cli %}.
+1. To edit a release, use the `gh release edit` subcommand. Replace `TAG` with the tag representing the release you wish to edit. For example, to edit a given release's title, use the following code:
+
+   ```shell
+   gh release edit TAG -t "New title"
```
Comment (sabrowning1) [inline content/repositories/releasing-projects-on-github/managing-releases-in-a-repository.md:85]:
```suggestion
   gh release edit TAG -t "NEW-TITLE"
```

File: content/repositories/releasing-projects-on-github/managing-releases-in-a-repository.md [hunk 5]
```diff
@@ -79,7 +79,13 @@ If you @mention any {% data variables.product.product_name %} users in the notes
 
 {% cli %}
 
-Releases cannot currently be edited with {% data variables.product.prodname_cli %}.
+1. To edit a release, use the `gh release edit` subcommand. Replace `TAG` with the tag representing the release you wish to edit. For example, to edit a given release's title, use the following code:
```
Comment (sabrowning1) [inline content/repositories/releasing-projects-on-github/managing-releases-in-a-repository.md:82]:
```suggestion
1. To edit a release, use the `gh release edit` subcommand. Replace `TAG` with the tag representing the release you wish to edit. For example, to edit the title for a release, use the following code, replacing `NEW-TITLE` with the updated title:
```

**Review by sabrowning1 — COMMENTED** [review]
Apologies @timsearle, I missed updating one other placeholder 🤦🏼‍♂️ I'll go ahead and commit my suggestions, then get this merged. Thanks for your quick response, and for your help keeping our docs up-to-date! ✨

**Review by sabrowning1 — APPROVED** [review]

**Pull request merged**